src/elements/content-sidebar/activity-feed/comment/__tests__/BaseComment.test.js

```diff
@@ -0,0 +1,273 @@
+// @flow
+
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { fireEvent, render, screen } from '@testing-library/react';
+import BaseComment from '../BaseComment';
+
+jest.mock('../../Avatar', () => () => 'Avatar');
+jest.mock('react-intl', () => ({
+    ...jest.requireActual('react-intl'),
+    FormattedMessage: ({ defaultMessage }: { defaultMessage: string }) => <span>{defaultMessage}</span>,
+}));
+
+const getUserProfileUrl = jest.fn().mockResolvedValue('https://www.test.com/');
+
+const currentUser = {
+    name: 'testuser',
+    id: 11,
+};
+
+const approverSelectorContacts = [];
+const mentionSelectorContacts = [];
+const TIME_STRING_SEPT_27_2017 = '2017-09-27T10:40:41-07:00';
+const TIME_STRING_SEPT_28_2017 = '2017-09-28T10:40:41-07:00';
+const comment = {
+    created_at: TIME_STRING_SEPT_27_2017,
+    tagged_message: 'test',
+    created_by: { name: '50 Cent', id: 10 },
+    permissions: { can_delete: true, can_edit: true, can_resolve: true },
+};
+
+const allHandlers = {
+    tasks: {
+        edit: jest.fn(),
+    },
+    contacts: {
+        getApproverWithQuery: jest.fn(),
+        getMentionWithQuery: jest.fn(),
+    },
+};
+const getWrapper = props =>
+    render(
+        <IntlProvider locale="en">
+            <BaseComment
+                id="123"
+                {...comment}
+                approverSelectorContacts={approverSelectorContacts}
+                currentUser={currentUser}
+                handlers={allHandlers}
+                mentionSelectorContacts={mentionSelectorContacts}
+                getUserProfileUrl={getUserProfileUrl}
```

Comment:
since this is an optional prop and we're not adding other functions like `getAvatarUrl` and `getMentionWithQuery` here, is it okay to remove this here and pass it in `getWrapper` in the test (`should render commenter as a link`) when it's needed in?